Create 2024-05-21-5xx-error-messages.md (#9011)
pantheon-systems/documentation: source/releasenotes/2024-05-21-5xx-error-messages.md

---
title: Unbranded 5xx error messages launched
published_date: "2024-05-21"
categories: [general]
---

> We’ve replaced all 5xx error messages with plain HTML and removed Pantheon branding for sites hosted on Pantheon. 

Reviewer: [](url)
```suggestion
We’ve replaced with Pantheon branding on [all platform-generated 5xx error messages](guides/errors-and-server-responses/5xx-errors) with plain HTML. 
```

Reviewer 2: typo, needs a leading `/` for the link to work

Reviewer: fixed

## Why this change

> Many of our customers, particularly those who are clients of our agency partners, may not be aware that their sites are hosted on Pantheon. For these clients, encountering Pantheon branded error messages could lead to confusion.

Reviewer: ```suggestion
Many of our customers, particularly those who are clients of our agency partners, may not be aware that their sites are hosted on Pantheon. For these clients, encountering Pantheon-branded error messages led to confusion.
```

Author: Thanks for reviewing. Good to go. @stevector  @rachelwhitton
